src/shared/src/types/change-request-types.ts

```diff
@@ -129,9 +129,9 @@ export interface ProjectProposedChangesCreateArgs extends WBSProposedChangesCrea
 
 export interface WorkPackageProposedChangesCreateArgs extends WBSProposedChangesCreateArgs {
   duration: number;
-  startDate: string;
-  stage: WorkPackageStage | null;
-  blockedBy: WbsNumber[];
+  startDate: Date;
+  stage: string;
+  blockedBy: string[];
   expectedActivities: string[];
   deliverables: string[];
 }
```

Comment:
this is the big thing, WorkPackageProposedChangesCreateArgs shouldn't extend the WBS one. Delete the WBS one and make these create args this:
```
  name: string;
  duration: number;
  startDate: string;
  stage?: WorkPackageStage;
  blockedBy: WbsNumber[];
  expectedActivities: string[];
  deliverables: string[];
  projectLeadId?: number;
  projectManagerId?: number;
```

Reply:
this is going to require you to update a couple places in the backend/ frontend to get the types (particularly undefined stuff) to play nice

Reply:
you should also update the workPackageProposedChanges validation stuff to reflect this